### DISCUSSION TITLE
[@types/gtag.js] Missing 'consent'

### DISCUSSION THREAD

**Discussion by WillSquire** [DefinitelyTyped/DefinitelyTyped#53958] [Issues with a @types package]
Types are missing the `consent` config options: https://developers.google.com/gtagjs/devguide/consent#adjusting_tag_behavior

@jablko @Rokt33r

**Comment by ediblecode** [comment]
Do you mean the consent modes defined here:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/131bd0ac19d20edbdfc20a6dbaa1ba44a5d10b99/types/gtag.js/index.d.ts#L110-L111

Are missing the following consent types:

- functionality_storage
- personalization_storage
- security_storage

As listed here: https://support.google.com/analytics/answer/9976101
